clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -588,9 +586,7 @@ public void onFailure(RuntimeException e) {
     private RequestFuture<Map<TopicPartition, OffsetAndTimestamp>> sendListOffsetRequest(final Node node,
                                                                                          final Map<TopicPartition, Long> timestampsToSearch,
                                                                                          boolean requireTimestamp) {
-        ListOffsetRequest.Builder builder = new ListOffsetRequest.Builder().
-                setTargetTimes(timestampsToSearch).
-                setReplicaId(ListOffsetRequest.CONSUMER_REPLICA_ID);
+        ListOffsetRequest.Builder builder = new ListOffsetRequest.Builder().setTargetTimes(timestampsToSearch);
```

Comment:
`CONSUMER_REPLICA_ID` is the default. We only have to pass it when using the builder from brokers.